test/elm/clinical/data.cql

```diff
@@ -37,13 +52,10 @@ define WrongShortCode: Code { code: 'M' } in "Female"
 define WrongMediumCode: Code { code: 'F', system: '3.16.840.1.113883.18.2' } in "Female"
 define LongCodeDifferentVersion: Code { code: 'F', system: '2.16.840.1.113883.18.2', version: 'HL7V2.6' } in "Female"
 define NullCode: Code { code: null, system: '2.16.840.1.113883.18.2', version: 'HL7V2.6' } in "Female"
```

Comment:
```suggestion
define NullCode: Code { code: null, system: '2.16.840.1.113883.18.2', version: 'HL7V2.6' } in "Female"
define NullCastAsCode: null as Code in "Female"
define NullCastAsCodeWithNullVS: null as Code in null as ValueSet
```
Need another two tests to make sure the `null as Code` and `null as ValueSet` cases are covered.

Reply:
added in [63a8e73](https://github.com/cqframework/cql-execution/pull/341/commits/63a8e73a4de4ebcf675127bca4af4cba03928af7)